bot/seasons/easter/egg_facts.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+
+class EasterFacts(commands.Cog):
+    """A cog for easter egg facts."""
+    def __init__(self, bot):
+        self.bot = bot
+        self.facts = self.load_json()
+
+    @staticmethod
+    def load_json():
+        """Loading the json data"""
```

Comment:
I think this docstring doesn't add much value to the current method name. How about adding what specifically the data contains? Something along the lines of
```suggestion
        """Load a list of easter egg facts from the resource JSON file."""
```

Reply:
And a space after the class string as well. (in the spirit of consistency.